### PR TITLE
Add date-of-death props and consistently named photo props

### PR DESCRIPTION
**Pull request by invisig0th**

### CODE REVIEW

File: synapse/models/person.py [hunk 1]
```diff
@@ -45,9 +45,16 @@ def getModelDefs(self):
...
                 ('ps:person', {}, (
                     ('dob', ('time', {}), {
-                        'doc': 'The Date of Birth (DOB) if known.',
+                        'doc': 'The date on which the person was born.',
+                    }),
+                    ('dod', ('time', {}), {
+                        'doc': 'The date on which the person died.',
                     }),
                     ('img', ('file:bytes', {}), {
+                        'deprecated': True,
```
Comment (vEpiphyte) [inline synapse/models/person.py:54]: ps:persona:img is not marked as deprecated

File: synapse/models/person.py [hunk 2]
```diff
@@ -45,9 +45,16 @@ def getModelDefs(self):
                 )),
                 ('ps:person', {}, (
                     ('dob', ('time', {}), {
-                        'doc': 'The Date of Birth (DOB) if known.',
+                        'doc': 'The date on which the person was born.',
+                    }),
+                    ('dod', ('time', {}), {
```
Comment (vEpiphyte) [inline synapse/models/person.py:50]: ps:persona is missing this dod property

**Review by jnwatson — APPROVED** [review]

File: synapse/models/person.py (hunk 2 above)
Comment (invisig0th) [reply synapse/models/person.py:50]: Purposely left this out due to an up-coming discussion where we may deprecate the entire `ps:persona`

File: synapse/models/person.py (hunk 1 above)
Comment (invisig0th) [reply synapse/models/person.py:54]: Also left off due to likely deprecating the whole form

**Pull request merged**